Troubleshooting.md

```diff
@@ -0,0 +1,210 @@
+We've listed all the known issues with the CORTX software and the steps to resolve them below. Please refer to our [support documentation](SUPPORT.md) to report new issues or want us to document steps to resolve an issue. 
+
+# Known Issues
+
+<details>
+<summary>Firmware bundle upload and update functionality not working</summary>
+<p>
+
+### Symptoms and causes
+  
+  1. Failed to fetch update status.
+  2. Failed to upload firmware bundle file.
+  3. Failed to update the firmware.
+
+### Resolution
+
+  1. Failed to fetch update status.
+    a. Refresh CMP UI after 30 secs.
+  2. Failed to upload firmware bundle file.
+    a. File format is incorrect. Please upload .bin file only.
+    b. File size is too large. Max file allowed is 2 GB.
+  3. Failed to update the firmware.
+    a. Failure reason as per return by provisioner component. Please check the csm_agent and provisioner logs.
+    
+ </p>
+ </details>
+ 
+<details>
+<summary>Software package upload and update functionality not working</summary>
+<p>
+
+### Symptoms and causes
+
+1. Failed to fetch update status.
+2. Failed to upload software bundle file.
+3. Failed to update the software.
+
+### Resolution
+
+1. Failed to fetch update status.
+  a. Refresh CMP UI after 30 secs.
+2. Failed to upload software bundle file.
+  a. File format is incorrect. Please upload .iso file only.
+  b. File size is too large. Max file allowed is 2 GB.
+3. Failed to update the software.
+  a. Failure reason as per return by provisioner component. Please check the csm_agent and provisioner logs.
+  
+ </p>
+ </details>
+  
+<details>
+<summary>CMP UI is not working</summary>
+<p>
+
+### Symptoms and causes
+
+1. URL is incorrect.
```

Comment:
What is this?  What URL?  This document needs to be much more precise.  Something like, "During X, you may see some error Y.  In such a case, here is what to do."

Currently this document is very far from being useful.

Perhaps asking for this document was not a good idea.  All of these steps should be in the individual quick start guides.  I think that is the biggest problem.  We have a bunch of quick start guides and these instructions are all for different ones of them.  I think this whole document is a mistake.  

On the basis of what I've seen so far, I don't think this document will be useful.